designsafe/apps/api/filemeta/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from django.contrib.postgres.fields import JSONField
+
+
+class FileMetaModel(models.Model):
+    system = models.CharField(max_length=255)
+    path = models.CharField(max_length=4096)
+    data = models.JSONField()
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
```suggestion
    created = models.DateTimeField(default=timezone.now)
    last_updated = models.DateTimeField(auto_now=True)
```
Using a "default" for `created` will let us populate it with the dates from the existing metadata, instead of everything having a 2024 creation date.